### ISSUE TITLE
Performance

### ISSUE THREAD

**Issue by ajabri** [ruotianluo/Faster-RCNN-Densecap-torch#4]
Did you end up making this work?

**Comment by ruotianluo** [comment]
Not yet. I was busy working on other things. Probably would pick this up in several months. 😭

**Comment by as641651** [comment]
I dont think you are filtering out detections with score < 0.05? This will decrease fp and increase recall
